doc/specs/#120,121,119-Update-Uninstall-List.md

```diff
@@ -0,0 +1,230 @@
+---
+author: <first-name> <last-name> <github-id>/<email>
+created on: <yyyy-mm-dd>
+last updated: <yyyy-mm-dd>
+issue id: <github issue id>
+---
+
+# Spec Title
+
+## Abstract
+
+For our preview of the Windows Package Manager, the goal was to enable install of apps.  In order to be a package manager, the Windows Package Manager must be able to list all installed packages, communicate if there is an update ane uninstall apps.  
+
+This spec will cover the following commands: list, update, and uninstall.
+## Inspiration
+
+See abstract.  :)
+
+## Solution Design
+
+**List**
+**Update \ Upgrade**
+**Uninstall**
+
+### List
```

Comment:
Handling available updates with this would be similar to how `dotnet` handles packages updates, nice 👍  https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package